src/Components/Products/Products.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Products.scss';
+
+class Products extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            items: []
+        }
+    }
+    componentDidMount() {
+        fetch("http://makeup-api.herokuapp.com/api/v1/products.json?brand=maybelline")
+            .then((response) => {
+                return response.json();
+            })
+            .then((result) => {
+                this.setState({
+                    items: result,
+                });
+            });
+    }
```

Comment:
- imports order
- we dont use class components
- no API calls inside a component
- we use axois and it should be under services
- the service function call should only be handled in pages